### PR TITLE
UCS/SYS: Fix math macro definitions

### PR DESCRIPTION
**Pull request by dmitrygladkov**
## What

Fixes macro definitions in math.h

## Why ?

They violate UCX code style

### CODE REVIEW

File: src/ucs/sys/math.h [hunk 1]
```diff
@@ -24,18 +24,18 @@ BEGIN_C_DECLS
 #define UCS_GBYTE    (1ull << 30)
 #define UCS_TBYTE    (1ull << 40)
 
-#define ucs_min(a, b) \
+#define ucs_min(_a, _b) \
 ({ \
-    typeof(a) _a = (a);  \
-    typeof(b) _b = (b);  \
-    _a < _b ? _a : _b;   \
+    typeof(_a) __a = (_a); \
```
Comment (brminich) [inline src/ucs/sys/math.h]: typically we use var names without `_` prefix inside macro

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9082/ for details (Mellanox internal link).

File: src/ucs/sys/math.h (hunk 1 above)
Comment (dmitrygladkov) [reply src/ucs/sys/math.h]: Done

**Comment by dmitrygladkov** [comment]
> plz, fix `ucs_convert_flag` as well

Sure, fixed

File: src/ucs/sys/math.h [hunk 2]
```diff
@@ -24,18 +24,18 @@ BEGIN_C_DECLS
...
 
-#define ucs_max(a, b) \
+#define ucs_max(_a, _b) \
 ({ \
-    typeof(a) _a = (a);  \
-    typeof(b) _b = (b);  \
-    _a > _b ? _a : _b;   \
+    typeof(_a) a = (_a); \
+    typeof(_b) b = (_b); \
+    a > b ? a :b; \
```
Comment (brminich) [inline src/ucs/sys/math.h]: space after `:`

File: src/ucs/sys/math.h [hunk 3]
```diff
@@ -92,9 +92,9 @@ static inline double ucs_log2(double x)
 
 /**
  * Convert flags without a branch
- * @return 'newflag' oldflag is set in 'value', otherwise - 0
+ * @return '_newflag' if '_oldflag' is set in '_value', otherwise - 0
  */
-#define ucs_convert_flag(value, oldflag, newflag) \
+#define ucs_convert_flag(_value, _oldflag, _newflag) \
```
Comment (brminich) [inline src/ucs/sys/math.h]: need to update names inside macro as well

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9085/ for details (Mellanox internal link).

File: src/ucs/sys/math.h (hunk 2 above)
Comment (dmitrygladkov) [reply src/ucs/sys/math.h]: done

File: src/ucs/sys/math.h (hunk 3 above)
Comment (dmitrygladkov) [reply src/ucs/sys/math.h]: ops :) done

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9087/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9088/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9090/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9095/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
@yosefe 

ok to merge?

**Pull request merged**